CONTRIBUTING.md

```diff
@@ -85,6 +86,24 @@ You may also want to read
 [The Mysterious PHP RFC Process](https://blogs.oracle.com/opal/post/the-mysterious-php-rfc-process-and-how-you-can-change-the-web)
 for additional notes on the best way to approach submitting an RFC.
 
+## Technical resources
+
+There are a number of technical resources on php-src. Unfortunately, they are
+scattered across different websites, and often outdated. Nonetheless, they can
+provide a good starting point for learning about the fundamentals of the code
+base.
+
+* https://www.phpinternalsbook.com/
```

Comment:
```suggestion
* https://www.phpinternalsbook.com/ - internal documentation
  * [Zvals](https://www.phpinternalsbook.com/php7/zvals.html), [strings](https://www.phpinternalsbook.com/php7/internal_types/strings.html), [callables](https://www.phpinternalsbook.com/php7/internal_types/functions/callables.html)
  * [Memory management](https://www.phpinternalsbook.com/php7/memory_management.html)
  * [Extensions design](https://www.phpinternalsbook.com/php7/extensions_design.html)
```
This could be most difficult one: There are some things missing, most notably php7 classes and arrays. It's probably worth link complete parts. PHP 5 classes probably could get also linked - I'm not sure how out-of-date they are.

Reply:
I don't think this one is necessary, there are many useful pages in that document. The older parts are lower in the table of contents.

Reply:
It'd be nice to have a quick links to most needed pages, often there is need to find something specific, and this list could help with that, instead of requiring to open every listed resource to check if topic is covered there.

Reply:
Listing the pages seems a bit like "this is all that's useful", which isn't the case for this one.

Reply:
Last could be link to TOC, though I'm used to google/grep everything needed, so I don't have strong feelings about it.